Templating/Helper/CmfHelper.php

```diff
@@ -168,6 +168,19 @@ public function find($path)
     }
 
     /**
+     * Finds a document by path and locale.
+     *
+     * @param $path
+     * @param $locale
+     *
+     * @return null|object
+     */
+    public function findTranslation($path, $locale)
+    {
+        return $this->getDm()->findTranslation(null, $path, $locale);
```

Comment:
just wondering: it could be convenient if this also accepts a document in addition to the path. so that i could get the translation of a document i already have, without worrying about how to get the path from the document.

you would start with is_string($path) and if not see if its a managed document and use the dm to read the path of the document... but that can also be added later when somebody actually needs it.